src/components/Navbar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../App.css';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/books',
+      text: 'Books',
+    },
+    {
+      id: 2,
+      path: '/categories',
+      text: 'Categories',
+    },
+  ];
```

Comment:
- [OPTIONAL] This is not a requirement for this milestone, but I suggest you fix this. ⚒️ 

-  It would be better to fix the path for:

![image](https://user-images.githubusercontent.com/79876959/145689054-7587a6ad-07da-44d2-90ba-4e8c1972cc9e.png)

because when you click in Books. It redirects to `/books` where you don't have any content. The solution is to change the path of books to `/`